NewForce-at-Mountwest/bangazon-site-take-2-centaurs
# Db ticket 11(AKA ticket #4 Users can search for products to buy)

- Once logged in, a user can see a search field in the navigation bar.

- User can type full or partial name of a product

- Once user types in desired product they can either press 'Enter' key or click on the 'Search' button

- Once user has made gesture, a list of the desired products will populate on page

RELATED TICKETS

#4 Users can search for products to buy

STEPS TO TEST

git add and git commit what you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout db-ticket-11 to access my branch.
If you don't see the most recent version of their files, try git pull origin db-ticket-11 on your server on my branch and make sure everything looks okay.
When you're not logged in, you shouldn't see the 'Sell Something' affordance
Once you're logged in, type something in search field and press enter
Gaze upon and be awed by working functionality!

Checklist:
[] I have commented my code, particularly in hard-to-understand areas
[] I have made corresponding changes to the documentation
[]My changes generate no new errors
[] I have added tests that prove my fix is effective or that my feature works
[] New and existing unit tests pass locally with my changes
